### ISSUE TITLE
Autocomplete for function sqrt

### ISSUE THREAD

**Issue by ishaojunzhang** [James-Yu/LaTeX-Workshop#501]
### How to reproduce

1. 
2. 
3. 

### Log messages

<details>
<summary>LaTeX Workshop Output</summary>

</details>

<details>
<summary>Developer Tools Console</summary>

</details>

**Comment by jlelong** [comment]
You are right. Surprisingly `\sqrt` is defined twice: once as a command and once using the snippet `\sqrt{x}`. For some reason, the command version is chosen.
